src/components/Admin/AdminLocalisations.tsx

```diff
@@ -0,0 +1,211 @@
+    <div id="localisation-panel">
+      <ModalWarning {...showModalWarning} />
+      <Header as="h1">Localisation</Header>
+      <Header as="h4">Currently installed languages</Header>
+      <div className="flex-row">
+        <p className="smaller-text">Enabled</p>
+      </div>
+      {languageOptions.map((language, index) => (
+        <div
+          style={{ position: 'relative' }}
+          onMouseEnter={() => setHoverIndex(index)}
+          onMouseLeave={() => setHoverIndex(null)}
+          key={language.code}
+        >
+          <LanguageRow language={language} index={index} handleSelect={handleSelect} />
+          {hoverIndex === index && (
+            <Icon
+              name="times circle outline"
+              size="large"
+              className="clickable"
+              style={{ position: 'absolute', right: 4, top: -4, color: 'grey' }}
+              onClick={() => handleRemove(language)}
+            />
+          )}
+        </div>
+      ))}
+      <Header as="h5">{strings.LOCALISATION_EXPORT_MESSAGE + ':'}</Header>
+      <div className="flex-row-start-center" style={{ gap: 20 }}>
+        <Button primary content={strings.LABEL_EXPORT} onClick={handleExport} />
+        <Checkbox
+          checked={exportDisabled}
+          onChange={() => setExportDisabled(!exportDisabled)}
+          label={strings.LOCALISATION_INCLUDE_DISABLED}
+        />
+      </div>
+      <Header as="h5">{strings.LOCALISATION_IMPORT_MESSAGE}</Header>
+      <div className="flex-row-start-center" style={{ gap: 20 }}>
+        <input
+          type="file"
+          ref={fileInputRef}
+          hidden
+          name="file-upload"
+          multiple={false}
+          accept=".csv"
+          onChange={handleFileImport}
+        />
+        <Button
+          primary
+          content={strings.LABEL_IMPORT}
+          onClick={() => fileInputRef?.current?.click()}
+        />
+        <Checkbox
+          checked={importDisabled}
+          onChange={() => setImportDisabled(!importDisabled)}
+          label={strings.LOCALISATION_INCLUDE_DISABLED}
+        />
+      </div>
+    </div>
+  )
+}
+
+const LanguageRow: React.FC<{
+  language: LanguageOption
+  handleSelect: (language: LanguageOption, index: number) => void
+  index: number
+}> = ({ language, handleSelect, index }) => {
+  return (
+    <div
+      className="flex-row-start-center"
+      style={{
+        gap: 20,
+        width: '90%',
+        backgroundColor: 'white',
+        marginBottom: 5,
+        padding: 10,
+        borderRadius: 8,
+      }}
+    >
+      <Checkbox checked={language.enabled} onChange={() => handleSelect(language, index)} />
+      <div className="flex-column">
+        <p style={{ marginBottom: 0 }}>
+          <strong>{language.languageName}</strong>
+        </p>
+        <p className="smaller-text">{language.description}</p>
+      </div>
+      <p style={{ fontSize: '2em', flexGrow: 1, textAlign: 'right' }}>{language.flag}</p>
+    </div>
```

Comment:
Would be feasible to move some of the styles to LESS files?

Reply:
Yeah, okay, have moved the general "row" style to LESS file. Left the very specific flag formatter inline.
4a317d3b